script/file_sizes.txt

```diff
@@ -1,5 +1,5 @@
-src/grammar.json    	0.2MB	     10984
-src/node-types.json 	0.1MB	      7677
-src/parser.c        	49.4MB	   1553316
+src/grammar.json    	0.3MB	     11864
+src/node-types.json 	0.1MB	      8495
+src/parser.c        	58.6MB	   1848207
 src/scanner.c       	0.0MB	        29
-total               	49.8MB	   1572006
+total               	59.0MB	   1868595
```

Comment:
I find it rather odd that adding field names has such a big impact on the parser size. @damieng do you have some insight why this is the case?

Reply:
@maxbrunsfeld may be able to answer this as well.